source/dotnet/Library/AdaptiveCards/AdaptiveDataQuery.cs

```diff
@@ -0,0 +1,67 @@
+using Newtonsoft.Json;
```

Comment:
missing copyright banner here and elsewhere

Reply:
added comment